### PR TITLE
Uniform and Laplace distributions

### PR DESCRIPTION
**Pull request by tdiethe**
*Issue #, if available:*

*Description of changes:*

Uniform and Laplace distributions + tests

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: mxfusion/components/distributions/__init__.py [hunk 1]
```diff
@@ -13,10 +13,11 @@
     random_gen
     univariate
     gp
+    laplace
```
Comment (meissnereric) [inline mxfusion/components/distributions/__init__.py]:
```suggestion
    laplace
    uniform
```

Comment (tdiethe) [reply mxfusion/components/distributions/__init__.py]: This is now complete. I also added tests to the gamma and normal distributions to match.

File: mxfusion/components/distributions/random_gen.py [hunk 2]
```diff
@@ -126,6 +134,66 @@ def sample_gamma(alpha=1, beta=1, shape=None, dtype=None, out=None, ctx=None, F=
...
+        :param out: output variable
+        :param ctx: execution context
+        :param F: MXNet node
+        :return: Array of samples
+        """
+        F = get_default_MXNet_mode() if F is None else F
+
+        samples = MXNetRandomGenerator._sample_univariate(
+            func=F.random.uniform, shape=shape, dtype=dtype, out=out, ctx=ctx, F=F)
+        # samples = F.broadcast_add(F.broadcast_mul(samples, F.broadcast_sub(high, low)), low)
```
Comment (meissnereric) [inline mxfusion/components/distributions/random_gen.py]: Remove commented code?

File: testing/components/distributions/laplace_test.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+            assert get_num_samples(mx.nd, log_pdf_rt) == num_samples
+        if np.issubdtype(dtype, np.float64):
+            rtol, atol = 1e-7, 1e-10
+        else:
+            rtol, atol = 1e-4, 1e-5
+        assert np.allclose(log_pdf_np, log_pdf_rt.asnumpy(), rtol=rtol, atol=atol)
+
+    @pytest.mark.parametrize(
+        "dtype, location, location_is_samples, scale, scale_is_samples, rv_shape, num_samples", [
+            # (np.float64, np.random.rand(1), False, np.random.rand(1) + 0.1, False, (3, 1), 10000),
```
Comment (meissnereric) [inline testing/components/distributions/laplace_test.py]: Remove?

**Comment by meissnereric** [comment]
@tdiethe Are you planning to finish up this and the Wishart PR? No hurry, just checking in.

**Comment by tdiethe** [comment]
Sorry thought they were all done ages ago!

**Pull request merged**